src/components/fields/checkbox-group/checkbox-group.tsx

```diff
@@ -158,12 +151,13 @@ export const CheckboxGroup = (props: IGenericFieldProps<TCheckboxGroupSchema>) =
 								checked={isCheckboxChecked(option.value)}
 								onChange={() => handleChange(option.value, option.none)}
 								error={!!error?.message}
-								subLabel={() =>
-									option.children ? (
-										<ToggleSublabel>
-											<Wrapper>{option.children}</Wrapper>
-										</ToggleSublabel>
-									) : null
+								compositeSection={
+									option.children
+										? {
+												children: <Wrapper>{option.children}</Wrapper>,
+												collapsible: false,
+										  }
+										: null
```

Comment:
help me disable the indent linting rule for this component and the radio button

```suggestion
									option.children
										? /* eslint-disable indent */
										  {
												children: <Wrapper>{option.children}</Wrapper>,
												collapsible: false,
										  }
										: /* eslint-enable indent */
										  null
```

Reply:
since prettier and eslint conflict, could the indent rule be disabled? the linter still flags out any formatting mismatch

Reply:
yeah that's fine too